python/interpret_community/common/constants.py

```diff
@@ -281,12 +281,6 @@ class Extension(object):
     LOCAL = 'local'
 
 
-class SHAPDefaults(object):
```

Comment:
This seems to be a public constant. Removing this might be a breaking change on the code using this constant. Could we keep this constant?

Reply:
done, added back constant